app/views/firms/partials/_header.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="firm__header">
   <%= heading_tag(@firm.name, level: 2, class: 'firm__name') %>
-  <% if search_form.face_to_face? %>
-    <%= render partial: 'search/partials/firm/adviser_distance', locals: { distance: @firm.closest_adviser }%>
+  <% if @firm.closest_adviser_distance.present? %>
+    <%= render partial: 'search/partials/firm/adviser_distance', locals: { distance: @firm.closest_adviser_distance }%>
```

Comment:
There are several partials that use instance variables in their markup. Would be clearer/easier to follow if we pass in locals instead.

Reply:
Good call. TBH I did not spend much time refactoring the views, there was a mixed use of locals/instance variable that's definitely time to address.